packages/nextjs/app/builders/0xd4A9f1f16c971f68FeF19882CBC019E616A85095/page.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+const ProfilePage: React.FC = () => {
```

Comment:
We use `NextPage` for this (instead of React.FC)

You can check how the other pages are created!

Reply:
Done! ✅